### ISSUE TITLE
AttributeError in DBClearableFileInput

### ISSUE THREAD

**Issue by asackofwheat** [victor-o-silva/db_file_storage#45]
With the changes added in 0.4.4, we were getting the following `AttributeError` when using `DBClearableFileInput`:

```
File "db_file_storage/form_widgets.py", line 35, in get_context
context['widget']['display'] = get_link_display(value.url)
AttributeError: 'NoneType' object has no attribute 'url'
```

The error occurs when trying to use this widget on a form with no values. We have fixed this on our end by creating a subclass of `DBClearableFileInput` that does a basic check for `value` when it is `None`, but we wanted to point it out here as well.

Let us know if you need additional details. We appreciate the work you're doing here. Thanks!

**Comment by victor-o-silva** [comment]
Thanks a lot for the report, @asackofwheat. Much appreciated!

Fixed in [v0.4.6](https://github.com/victor-o-silva/db_file_storage/releases/tag/0.4.6) and available [on PyPI](https://pypi.python.org/pypi/django-db-file-storage/0.4.6).

It should work for you out-of-the-box now. Please give it a try with 0.4.6.

**Comment by asackofwheat** [comment]
Thanks for making this change. The fix works on our end!
